connector_helpscout/models/helpscout_folder/common.py

```diff
@@ -30,6 +30,11 @@ class HelpScoutFolder(models.Model):
         string='HelpScout Folder Name',
         required=True,
     )
+    task_ids = fields.One2many(
```

Comment:
Typically in bind models, we will want to join with other bind models. This keeps a consistency that is otherwise really hard to mentally picture.